src/sentry/api/serializers/models/team.py

```diff
@@ -385,14 +386,25 @@ def get_attrs(
         self, item_list: Sequence[Team], user: Any, **kwargs: Any
     ) -> Mapping[Team, MutableMapping[str, Any]]:
 
-        result: MutableMapping[Team, MutableMapping[str, Any]] = {team: {} for team in item_list}
+        result: MutableMapping[int, MutableMapping[str, Any]] = {
+            team.id: ({"members": []} if "members" in self.expand else {}) for team in item_list
+        }
+        teams_by_id: Mapping[int, Team] = {t.id: t for t in item_list}
 
-        if "members" in self.expand:
-            member_map = get_scim_teams_members(item_list)
-            for team in item_list:
-                # if there are no members in the team, set to empty list
-                result[team]["members"] = member_map.get(team, [])
-        return result
+        if teams_by_id and "members" in self.expand:
+            org_id: int = next(iter(teams_by_id.values())).organization_id
+            team_ids: List[int] = [t.id for t in item_list]
+            team_memberships: List[RpcTeamMembership] = organization_service.get_team_memberships(
```

Comment:
I'm assuming we need to serialize teams from control silo? :cry:

Reply:
Ah right, this is an artifact that I can remove from when I was trying to resolve SCIM control handling.  I will inline this logic.